### DISCUSSION TITLE
Setting threshold for PRs (Pull Requests)

### DISCUSSION THREAD

**Discussion by duyn9uyen** [coverlet-coverage/coverlet#1309] [Q&A] [answered]
Is it possible to set the threshold flag (/p:Threshold={number}) for pull requests and not for the entire project?

**Comment by MarcoRossignoli** [comment, marked as answer]
@duyn9uyen can you open an issue for this question? https://github.com/coverlet-coverage/coverlet/issues

Anyway coverlet doesn't know in which context is running, so I think that maybe you can in your pipeline change that parameter using a variable knowing the current context(PR or not).

**Comment by Bertk** [comment]
Closing this discussion since it’s no longer relevant to the current state of the project.
